optima/programs.py

```diff
@@ -531,7 +531,7 @@ def getpars(self, coverage, t=None, parset=None, results=None, ind=0, perturb=Fa
                         if die:
                             raise OptimaException(errormsg)
                         else:
-                            printv(errormsg, 1, verbose)
+                            printv(errormsg, 3, verbose) # WARNING, not sure how serious this is...feels like it shouldn't happen
```

Comment:
@robynstuart can you check this? originally i had it give a fairly high-priority error because i didn't think a parameter should ever get out of bounds even with 100% coverage...but if it's ok then you can probably leave with verbose=3.